src/components/sections/Header.js

```diff
@@ -1,68 +1,42 @@
 import React from 'react';
 import styled from 'styled-components';
-import { StaticQuery, graphql } from 'gatsby';
-import Img from 'gatsby-image';
 
 import { Container } from '@components/global';
-import ExternalLink from '@common/ExternalLink';
+
+import { ReactComponent as GraaspLogo } from '@images/art/hand.svg';
 
 const Header = () => (
-  <StaticQuery
-    query={graphql`
-      query {
-        art_build: file(
-          sourceInstanceName: { eq: "art" }
-          name: { eq: "build" }
-        ) {
-          childImageSharp {
-            fluid(maxWidth: 1400) {
-              ...GatsbyImageSharpFluid_withWebp_tracedSVG
-            }
-          }
-        }
-      }
-    `}
-    render={data => (
-      <HeaderWrapper>
-        <Container>
-          <Grid>
-            <Art>
-              <Img fluid={data.art_build.childImageSharp.fluid} />
-            </Art>
-            <Text>
-              <h1>
-                Fast in
-                <br />
-                every way
-                <br />
-                that matters
-              </h1>
-              <br />
-              <p>
-                <StyledExternalLink href="https://github.com/ajayns/gatsby-absurd">
-                  Check out source &nbsp;&#x2794;
-                </StyledExternalLink>
-              </p>
-            </Text>
-          </Grid>
-        </Container>
-      </HeaderWrapper>
-    )}
-  />
+  <HeaderWrapper>
+    <Container>
+      <Grid>
+        <Art>{GraaspLogo()}</Art>
+        <Text>
+          <h1>
+            Graasp is a non-profit organization promoting innovation in digital
```

Comment:
I'm a big fan of capitalisation for these sort of titles/captions. I.e. "Graasp is a Non-Profit Organization Promoting Innovation in Digital Education". Otherwise, I would add a full stop at the end. Very similar to how ProtonMail does things.